root/static/styles/widgets.less

```diff
@@ -373,10 +385,19 @@ div.autocomplete2 {
                 font-weight: bold;
                 text-align: center;
             }
+            &.level1 { padding-left: 12px; }
+            &.level2 { padding-left: 20px; }
+            &.level3 { padding-left: 28px; }
+            &.level4 { padding-left: 36px; }
         }
     }
 }
 
+label.autocomplete2-label {
```

Comment:
I filed https://tickets.metabrainz.org/browse/MBS-14001, but I think that this selector is less specific than a `table.row-form label` selector that's setting `display: inline-block;`.